package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "scripts": {
+    "format": "fvm dart format -l 120 .",
+    "analyze": "fvm dart analyze",
+    "fix": "fvm dart fix --apply",
+    "fix:dry": "fvm dart fix --dry-run"
+  }
+}
```

Comment:
Not sure if it is good style to add a package.json to a flutter project for commands only. Let me know if you have any better ideas, but I think its especially helpful for the first command since apparently its not possible to globally set the line length.